package.json

```diff
@@ -64,6 +69,7 @@
     "build:docker:single:image": "docker build -f hosting/single/Dockerfile -t budibase:latest .",
     "build:docker:single": "npm run build:docker:pre && npm run build:docker:single:image",
     "build:docker:dependencies": "docker build -f hosting/dependencies/Dockerfile -t budibase/dependencies:latest ./hosting",
+    "build:docker:run": "yarn build && docker-compose -f hosting/docker-compose.build.yaml up --build",
```

Comment:
Alternative: `dev:docker`

Reply:
I like it, easier to understand and remember